src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -85,24 +88,24 @@ private set
         /// The task which computes the main rename locations against the original workspace
         /// snapshot.
         /// </summary>
-        private JoinableTask<IInlineRenameLocationSet> _allRenameLocationsTask;
+        private Task<IInlineRenameLocationSet> _allRenameLocationsTask;
 
         /// <summary>
         /// The cancellation token for most work being done by the inline rename session. This
         /// includes the <see cref="_allRenameLocationsTask"/> tasks.
         /// </summary>
-        private readonly CancellationTokenSource _cancellationTokenSource = new CancellationTokenSource();
+        private readonly CancellationTokenSource _cancellationTokenSource = new();
 
         /// <summary>
         /// This task is a continuation of the <see cref="_allRenameLocationsTask"/> that is the result of computing
         /// the resolutions of the rename spans for the current replacementText.
         /// </summary>
-        private JoinableTask<IInlineRenameReplacementInfo> _conflictResolutionTask;
+        private Task<IInlineRenameReplacementInfo> _conflictResolutionTask;
```

Comment:
⚠️ This was previously adhering to the JTF _requirement_ that this be `JoinableTask<T>` and not `Task<T>`. Is there a reason we are changing the code to no longer follow patterns?